bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/thermostat.xml

```diff
@@ -85,6 +89,10 @@
 		<label>Outdoor Temperature</label>
 		<description>The current outdoor temperature detected by this thermostat.</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
```

Comment:
I would question this one as I believe a better way forward would be to move to using system.outdoor-temperature See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types and then you allow the framework to handle the tags for this.

Reply:
I have it as that already but as of now the UI does not take advantage of that in displaying it to the user in the locations page.

Reply:
You must be using 3.1 Stable or older, as of 3.2 Milestone 1 I can confirm that the tags are added by the core as I tested it just now and I saw the PR get merged.
Just a tip it needs to be a POINT added directly in the semantic model under the LOCATION, it can not be inside an EQUIPMENT group for it to display.

Reply:
> Just a tip it needs to be a POINT added directly in the semantic model under the LOCATION, it can not be inside an EQUIPMENT group for it to display.

That would probably be why mine doesn't display